msteams-platform/toolkit/debug-local.md

```diff
@@ -156,15 +157,15 @@ Launches the debug configurations defined in `.vscode/launch.json` as follows:
 
 :::image type="content" source="../assets/images/teams-toolkit-v2/debug/launch-debuggers.png" alt-text="Launch debugger":::
 
-The following table lists the debug configuration names and types for project with tab app and bot app:
+The debug configuration names and types for project with Tab app and Bot app:
```

Comment:
Please rephrase to the previous sentence:
The following table lists the debug configuration names and types for project with tab app and bot app: